CONTRIBUTING.md

```diff
@@ -5,7 +5,10 @@ These are mostly guidelines, not rules.
 Use your best judgement, and feel free to propose changes to this document in a pull request.
```

Comment:
We can add more contributing things here later. I left most of that out on purpose in the first version and focused on the engineering process.